src/main/resources/templates/index.html

```diff
@@ -34,7 +34,7 @@ <h2 th:text="|Week ${currentWeek.getWeekNumberNumber()}|"></h2>
     </header>
 
     <main>
-        <div class="task" th:each="task : ${currentWeek.getSortedTasks()}" th:object="${task}" th:data-priority="*{priority}" th:data-id="*{id}" th:data-done="*{done}">
+        <div class="task" th:each="task : ${currentWeek.getSortedTasks()}" th:object="${task}" th:data-priority="*{priority}" th:data-id="*{id}" th:data-done="*{done}" id="task-input">
```

Comment:
id task-input should be renamed or deleted, if the targeting using a class is possible